pages/_app.tsx

```
@@ -37,6 +37,21 @@ function MyApp({ Component, pageProps }: AppProps) {
           content="Easily add collaboration to your apps with our API-based services."
         />
       </Head>
+      <script
+        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)